packages/repository-tests/src/crud/freeform-properties.suite.ts

```diff
@@ -18,38 +17,59 @@ import {
   CrudTestContext,
   DataSourceOptions,
 } from '../types.repository-tests';
+import {assert} from 'console';
 
 export function freeformPropertiesSuite(
   dataSourceOptions: DataSourceOptions,
   repositoryClass: CrudRepositoryCtor,
   features: CrudFeatures,
 ) {
-  skipIf<[(this: Suite) => void], void>(
-    !features.freeFormProperties,
-    describe,
-    'free-form properties (strict: false)',
-    () => {
-      before(deleteAllModelsInDefaultDataSource);
+  @model({settings: {strict: false}})
+  class Freeform extends Entity {
+    @property({
+      type: features.idType,
+      id: true,
+      description: 'The unique identifier for a product',
+    })
+    id: MixedIdType;
+
+    @property({type: 'string', required: true})
+    name: string;
 
-      @model({settings: {strict: false}})
-      class Freeform extends Entity {
-        @property({
-          type: features.idType,
-          id: true,
-          description: 'The unique identifier for a product',
-        })
-        id: MixedIdType;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    [propName: string]: any;
 
-        @property({type: 'string', required: true})
-        name: string;
+    constructor(data?: Partial<Freeform>) {
+      super(data);
+    }
+  }
 
-        // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        [propName: string]: any;
+  it('SQL DBs should warn that {strict: false} mode is not supported', async () => {
+    before(function() {
+      sinon.spy(console, 'warn');
+    });
+    withCrudCtx(async (ctx: CrudTestContext) => {
+      new repositoryClass(Freeform, ctx.dataSource);
+    });
+    /* istanbul ignore if */
+    if (!features.freeFormProperties) {
+      //eslint-disable-next-line no-unused-expressions
+      expect(console.warn).to.be.called;
```

Comment:
This should be `expect(console.warn).to.be.called();` (would also get rid of the eslint thing)